ACVM/ViewController.swift

```diff
@@ -154,6 +175,15 @@ class ViewController: NSViewController, FileDropViewDelegate {
         return adjustedRamText
     }
     
+    private var displayAdaptor: String {
+        let adjustedDisplayText = graphicPopupButton.titleOfSelectedItem;
+        guard adjustedDisplayText != nil else {
+            return "ramfb"
+        }
+        
+        return adjustedDisplayText!
```

Comment:
Let's do `guard let` instead of force unwrap ^^

```
guard let adjustedDisplayText = graphicPopupButton.titleOfSelectedItem else {
    return "ramfb"
}
return adjustedDisplayText
```